src/components/neumorphic/ConfigurationRow.tsx

```diff
@@ -1,5 +1,18 @@
-import PropTypes from 'prop-types';
+import { ChangeEvent } from 'react';
 import { camelize } from '../../utils';
+
+interface ConfigurationRowProps {
+  className?: string;
+  label: string;
+  type: string;
+  value: string | number;
+  min?: string | number;
+  max?: string | number;
+  step?: string | number;
```

Comment:
Deshaceste de todos los string que no sean necesarios, ya solo deja number si es posible
```suggestion
  value: number;
  min?: number;
  max?: number;
  step?: number;
```